main.tf

```diff
@@ -7,6 +7,14 @@
  */
 
 locals {
+  drupal_network_policy_list = [
+    for p in var.drupal_projects_list : {
```

Comment:
I would prefer to use the `local.namespace_list` to automatically keep in sync the namespace name creation and avoid code/logic duplication.